task-launcher/src/tasks/shared/helpers/getTranslations.ts

```diff
@@ -53,11 +53,15 @@ export const getTranslations = async (configLanguage?: string) => {
   }
 
   async function fetchData() {
-    const urls = [
-      // This will eventually be split into separate files
-      `https://storage.googleapis.com/road-dashboard/item-bank-translations.csv`,
-    ];
-
+    // This will eventually be split into separate files
+    let urls;
+    isDev ? 
+      urls = [
+        `https://storage.googleapis.com/levante-dashboard-dev/item-bank-translations.csv`
+      ] :
+      urls = [
+      ` https://storage.googleapis.com/levante-dashboard-prod/item-bank-translations.csv`
+      ]
```

Comment:
This is a nit, but could we make this more succint with string interpolation?

`` https://storage.googleapis.com/levante-dashboard-${isDev ? 'dev' : 'prod'}/item-bank-translations.csv``